zzzz_modular_occulus/code/datum/craft/recipes/weapon.dm

```diff
@@ -0,0 +1,11 @@
+/*/datum/craft_recipe/weapon/stungloves
```

Comment:
Why's this commented out!

Reply:
I originally tried making the recipe modular, but it’s not possible do so. I forgot to delete this file >..>